src/helpers/domainFindOptionsHelper.ts

```diff
@@ -0,0 +1,136 @@
+import { HttpRequest } from "@azure/functions";
+import { SortDirection } from "mongodb";
+import { DomainFindOptions } from "@zero-tech/data-store-core";
+import { DomainSortDirection, domainReflectionSchema } from "../types";
+declare type AscDesc = -1 | 1;
+declare type IncludeExlucde = 0 | 1;
```

Comment:
*Exclude

Reply:
![image](https://user-images.githubusercontent.com/25021249/174356680-ac819946-5aaf-4ee8-8cf6-09e36dfe3e19.png)

How did I type that?